### DISCUSSION TITLE
We need aw-watcher-vim (and also other watchers)?

### DISCUSSION THREAD

**Discussion by harrykar** [ActivityWatch/activitywatch#1019] [Q&A]
Few days ago I asked a question https://github.com/ActivityWatch/aw-watcher-vim/issues/31 on aw-watcher-vim's repo after have read the other few (3 IIRC) issues there, but seems to me aw-watcher-vim actually is a not so followed(maybe discontinued?) project

So I re-propose the question here (I'm sorry if that's redundant)
TIA

**Comment by 0xbrayo** [comment]
I don't use neovim, but Erik does and I'm sure the watcher works... Maybe try with neovim. If it works, you can try modifying it to work with your editor
